vision/cloud-client/src/main/java/com/example/vision/Detect.java

```diff
@@ -95,19 +95,19 @@ public static void main(String[] args) throws Exception, IOException {
   public static void argsHelper(String[] args, PrintStream out) throws Exception, IOException {
     if (args.length < 1) {
       out.println("Usage:");
-      out.printf(
-          "\tmvn exec:java -DDetect -Dexec.args=\"<command> <path-to-image>\"\n"
+      out.format(
+          "\tmvn exec:java -DDetect -Dexec.args=\"<command> <path-to-image>\"%n"
               + "\tmvn exec:java -DDetect -Dexec.args=\"ocr <path-to-file> <path-to-destination>\""
-              + "\n"
-              + "Commands:\n"
+              + "%n"
+              + "Commands:%n"
               + "\tfaces | labels | landmarks | logos | text | safe-search | properties"
-              + "| web | web-entities | web-entities-include-geo | crop | ocr \n"
-              + "| object-localization \n"
-              + "Path:\n\tA file path (ex: ./resources/wakeupcat.jpg) or a URI for a Cloud Storage "
-              + "resource (gs://...)\n"
-              + "Path to File:\n\tA path to the remote file on Cloud Storage (gs://...)\n"
-              + "Path to Destination\n\tA path to the remote destination on Cloud Storage for the"
-              + " file to be saved. (gs://BUCKET_NAME/PREFIX/)\n");
+              + "| web | web-entities | web-entities-include-geo | crop | ocr %n"
+              + "| object-localization %n"
+              + "Path:%n\tA file path (ex: ./resources/wakeupcat.jpg) or a URI for a Cloud Storage "
+              + "resource (gs://...)%n"
+              + "Path to File:%n\tA path to the remote file on Cloud Storage (gs://...)%n"
+              + "Path to Destination%n\tA path to the remote destination on Cloud Storage for the"
+              + " file to be saved. (gs://BUCKET_NAME/PREFIX/)%n");
```

Comment:
Why isn't this just a bunch of out.println(...);  ??

Reply:
I will just remove this argsHelper method